packages/fast-development-site-react/src/components/toc/toc-item.tsx

```diff
@@ -53,7 +53,7 @@ const style: ComponentStyles<ITocItemManagedClasses, IDevSiteDesignSystem> = {
         color: (config: IDevSiteDesignSystem): string => {
             return config.foregroundColor;
         },
-        boxShadow: `0 ${toPx(2)} ${toPx(4)} ${toPx(-1)} #ccc`,
+        boxShadow: `0 ${toPx(2)} ${toPx(4)} rgba(0, 0, 0, 0.06)`,
```

Comment:
It seems that we still have the issue that the drop shadow is visible over the canvas. What was this change meant to fix?

Reply:
The notes said to update to current design, so I simply used the box shadow data found in the design on Figma. This is how it looks now:
<img width="416" alt="screen shot 2018-07-11 at 9 01 19 pm" src="https://user-images.githubusercontent.com/22480660/42611993-a19f2730-854d-11e8-8871-ffba76176d1b.png">
Is this the expected rendering? Seems to match the Figma spec.

Reply:
Figma file does not have this shadow overlapping the canvas: 
<img width="316" alt="screen shot 2018-07-12 at 9 07 38 am" src="https://user-images.githubusercontent.com/7559015/42645513-1757fde8-85b3-11e8-9fac-46722f2088f5.png">